### ISSUE TITLE
Syncing tasks not working with Tine20

### ISSUE THREAD

**Issue by bernhard1401** [jobisoft/TbSync#40]
I want to sync Thunderbirds Contacts, Calendars and Tasks with Tine20. When I try to sync tasks I get an error. The Log tells me the following

```
[Exception... "Component returned failure code: 0x804a0104 [calIDateTime.icalString]"  nsresult: 
"0x804a0104 (<unknown>)"  location: "JS frame :: resource://calendar/modules/calUtils.jsm -> 
file:///Users/bernhard/Library/Thunderbird/Profiles/ahurds4v.default/extensions/%7Be2fda1a4-762b-
4020-b5ad-a41df1933103%7D/calendar-js/calUtils.js :: _calIcalCreator/< :: line 44"  data: no]  
(unbekannt)
	_calIcalCreator/< file:///Users/bernhard/Library/Thunderbird/Profiles/ahurds4v.default/extensions
/%7Be2fda1a4-762b-4020-b5ad-a41df1933103%7D/calendar-js/calUtils.js:44:13
	eas.sync.Tasks.setThunderbirdItemFromWbxml chrome://tbsync/content/provider/eas/tasksync.js:22:23
	eas.sync.requestRemoteChanges< chrome://tbsync/content/provider/eas/sync.js:516:29
	next self-hosted:1120:9
	TaskImpl_run resource://gre/modules/Task.jsm:319:42
	bound TaskImpl_run self-hosted:957:17

```
Thanks

**Comment by jobisoft** [comment]
can I please have the XML of that task? Or the entire debug.log (via email)?

**Comment by jobisoft** [comment]
probably fixed in 0.7.3-beta1

**Comment by jobisoft** [comment]
Please update your beta version of TbSync to [0.7.6.8](https://github.com/jobisoft/TbSync/releases/download/v0.7.6/TbSync-0.7.6.8.xpi) or any later version available at the [release page](https://github.com/jobisoft/TbSync/releases).

This version now includes an update notification, so I no longer have to contact all beta users, to update their TbSync versions:

https://raw.githubusercontent.com/jobisoft/TbSync/master/screenshots/update_notification.png

While on the beta release channel, I strongly suggest to update, if a newer version is available. Once a new stable version gets released, you automatically switch back to the stable release channel (you no longer receive update notifications for beta versions).
